doc/eip.md

```diff
@@ -12,7 +12,7 @@ requires: EIP-155, EIP-1014, EIP-7702
 
 ## Abstract
 
-This ERC specifies an alternative mechanism and deployment parameters for a permissionless CREATE2 factory contract with deterministic cross-chain address (`0xC0DE207acb0888c5409E51F27390Dad75e4ECbe7`) and code. The deployed contract can then be used to deploy any other contracts to deterministic addresses by leveraging the [EIP-1044](./eip-1044.md) `CREATE2 (0xf5)` opcode.
+This ERC specifies an alternative mechanism and deployment parameters for a permissionless CREATE2 factory contract with deterministic cross-chain address (`0xC0DE207acb0888c5409E51F27390Dad75e4ECbe7`) and code. The deployed contract can then be used to deploy any other contracts to deterministic addresses by leveraging the [EIP-1014](./eip-1014.md) `CREATE2 (0xf5)` opcode and the [EIP-7702](./eip-7702.md) `Set Code for EOAs` transaction type.
```

Comment:
I found this a bit confusing - as it implies that:

> The deployed contract can then be used to deploy any other contracts to deterministic addresses

Relies on EIP-7702, which isn't true. Only the deployment of the "deployed contract" does.

What do you think of:

```suggestion
This ERC specifies an alternative mechanism using the [EIP-7702](./eip-7702.md) `Set Code for EOAs (0x4)` transaction type as well as deployment parameters for a permissionless CREATE2 factory contract with deterministic cross-chain address (`0xC0DE207acb0888c5409E51F27390Dad75e4ECbe7`) and code. The deployed contract can then be used to deploy any other contracts to deterministic addresses by leveraging the [EIP-1014](./eip-1014.md) `CREATE2 (0xf5)` opcode.
```